common.go

```diff
@@ -17,3 +18,20 @@ func debugLog(format string, args ...interface{}) {
 	}
 	log.Printf(format+"\n", args...)
 }
+
+//logURL replaces existing password from the given URL
```

Comment:
nit: Add a space after // and end the comment with punctuation.

https://golang.org/doc/effective_go.html#commentary